Instrukcja

```diff
@@ -32,6 +33,5 @@ Plan rundy:
 Akcje:
 1. Sklep - żywność
 2. Apteka - leki
-3. Śmietnik (testowo, to wywalenia)
```

Comment:
Nazwa brancha to "usunę las". Jak się to ma do śmietnika?

Reply:
pomyliłem, szybko pisząc, zaraz zmienie nazwa